service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/question/RespondentsRefusedBlockPrePopulator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.fpl.service.orders.prepopulator.question;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock;
+import uk.gov.hmcts.reform.fpl.model.order.selector.Selector;
+import uk.gov.hmcts.reform.fpl.service.RespondentsRefusedFormatter;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.model.order.selector.Selector.newSelector;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class RespondentsRefusedBlockPrePopulator implements QuestionBlockOrderPrePopulator {
+
+    private final RespondentsRefusedFormatter respondentsRefusedFormatter;
+
+    @Override
+    public OrderQuestionBlock accept() {
+        return OrderQuestionBlock.RESPONDENTS_REFUSED;
+    }
+
+    @Override
+    public Map<String, Object> prePopulate(CaseData caseData) {
+        final Selector respondentsRefusedSelector = newSelector(
+            caseData.getAllRespondents().size() + caseData.getAllOthers().size());
```

Comment:
is this for just respondents or respondents+others, might not need to add the size of the others?